src/rpc/RPC.cpp

```diff
@@ -223,8 +230,9 @@ static HandlerTable handlerTable{
     {"ledger", &doLedger, {}},
     {"ledger_data", &doLedgerData, LimitRange{1, 100, 2048}},
     {"nft_buy_offers", &doNFTBuyOffers, LimitRange{1, 50, 100}},
-    {"nft_info", &doNFTInfo},
+    {"nft_info", &doNFTInfo, {}, true},
     {"nft_sell_offers", &doNFTSellOffers, LimitRange{1, 50, 100}},
+    {"nft_tx", &doNFTTx, LimitRange{1, 50, 100}, true},
```

Comment:
can we call this `nft_txs`, or `account_nft_tx` or something that implies that this will return _multiple_ txs.

When I read `nft_tx`, I think `tx`, which only returns one transaction.

Reply:
well the idea is to mimic `account_tx`, which gets you transactions related to one account. this is the same but for one nft :)

Reply:
This is a tough one. I agree `nft_txs` is a better name, but it's also good to mimic `account_tx`. We could also just call it `nft_history` or something similar

Reply:
Up to y'all on this one.